Rules2020/32 ファウルと反則のセクションの再構成 (#30)
kkimurak/ssl-rules-jp: chapters/offenses.adoc

== 反則/Offenses

> === 試合の進展なし/No Progress In Game

Author: もうすこしマシな翻訳が欲しい気がします...

Author: 「試合の停滞」等で当てはめてみて、しっくり来たらこれにします。

If there is no progress in the game for 5 seconds (Division A) or 10 seconds (Division B) while both teams are allowed to
<<ボールの操作/Ball Manipulation, manipulate the ball>>, the game is <<停止/Stop, stopped>>
and continued by a <<フォーススタート/Forced Start, forced start>>.

=== ダブルタッチ/Double Touch

<<キックオフ/Kick-Off, キックオフ>>、<<フリーキック/Free Kick, フリーキック>>、<<ペナルティーキック/Penalty Kick, ペナルティーキック>>によってボールがインプレイに持ち込まれたとき、キッカーは他のロボットが触れるか、試合が停止するまでボールに触れてはいけない。 +
When the ball is brought <<インプレイとアウトオブプレイ/Ball In And Out Of Play, into play>> following a <<キックオフ/Kick-Off, kick-off>> or <<フリーキック/Free Kick, free kick>>, the kicker is not allowed to touch the ball until it has been touched by another robot or the game has been stopped.

<<インプレイとアウトオブプレイ/Ball In And Out Of Play, インプレイ>>になったとみなされるには、少なくとも0.05m以上ボールが動いていなければいけない。 +
The ball must have moved at least 0.05 meters to be considered as <<インプレイとアウトオブプレイ/Ball In And Out Of Play, in play>>.

ダブルタッチによって試合は<<停止/Stop, 停止>>され、次いで同じボールの位置から<<フリーキック/Free Kick, フリーキック>>となる。 +
A double touch results in a <<停止/Stop, stop>> followed by a <<フリーキック/Free Kick, free kick>> from the same ball position.

> NOTE: キックを行おうとしている間に、ロボットが短い距離でボールを跳ねさせてしまう事がありうることが理解される。

Author: もう少しマシな翻訳が欲しいです...
こんな感じでしょうか?これでも微妙な気がしますが...

```suggestion
NOTE: キックを行おうとしている間に、ロボットが短い距離でボールを跳ねさせるのはあり得る事である。
```

Author: 全くマシではない気がしますが、ひとまず直してみました。

NOTE: 人間の主審は、各セクションに記述された罰則に関係なく、そのファウルが繰り返される場合は<<イエローカード/Yellow Card, イエローカード>>を、特に極端な場合は<<レッドカード/Red Card, レッドカード>>を即座に与えることを選択できる。 +
Regardless, of the prescribed penalties in this section, if a
foul is severe or repeated, the referee can choose to immediately
issue a <<Yellow Card, yellow card>> or in extreme cases a <<Red Card,
red card>>.

==== 試合の停止を伴うファウル/Stopping Fouls
このセクションにあるファウルは試合を<<停止/Stop, 停止>>させ、反則が発生した時点でボールが置かれていた位置からの<<フリーキック/Free Kick, フリーキック>>で再開する。 +
Fouls in this section cause the game to <<Stop, stop>> and then resume
with a <<Free Kick, free kick>> from the position where the ball was
located when the foul began happening.

===== ロボットの相手ディフェンスエリアへの極端な接近/Robot Too Close To Opponent Defense Area
<<試合の再開/Resuming The Game, 試合を再開する>>前の、<<停止/Stop, 停止>>、<<フリーキック/Free Kick, フリーキック>>の間、すべてのロボットは相手の<<ディフェンスエリア/Defense Area, ディフェンスエリア>>から少なくとも0.2m以上離れていなければならない。 +
During <<停止/Stop, stop>> and <<フリーキック/Free Kick, free kicks>>, before the ball <<試合の再開/Resuming The Game, has entered play>>, all robots have to keep at least 0.2 meters distance to the opponent <<ディフェンスエリア/Defense Area, defense area>>.

ロボットが相手ディフェンスエリアから離れるため、2秒間の猶予が設けられる。 +

> There is grace period of 2 seconds for the robot to move away from the opponent defense area.

Author: 原文から取りこぼしがありました。

```suggestion
There is a grace period of 2 seconds for the robots to move away from the opponent defense area.
```

===== プッシング/Pushing
あるロボットが相手のロボットに外力を加えて押していて、両方のロボットがボールもしくは互いに接触している(たとえば互いのロボットが相手のロボットの方向に移動している)時、これはプッシングの反則となる。 +
A robot pushes an opponent robot if both robots keep contact to the ball or to each other while the robot exerts force onto the opponent robot, such that both robots travel towards the opponent robot.

> NOTE: 両方のロボットが同じような力で互いに押し合っている場合は、どちらに対してもファウルはとられない。 +

Author: このプルリクエストでは対象としませんが、 #31 で次のように修正したいと思います。
```suggestion
NOTE: 両方のロボットが同じような力で互いに押し合っている場合は、どちらのチームもファウルとはならない。。 +
```

NOTE: (例えばねじなどの)金属パーツと大きな部品は一般的に潜在的に脅威をもたらし、非常に小さい(例えば小車輪のゴムなどの)非金属のパーツはそうではない。 +
Metal parts (screws for example) as well as larger parts generally pose a potential threat, very small non-metal parts (for example rubber subwheel rings) don't.

===== マルチプルディフェンス/Multiple Defenders

> NOTE: このルールは<<ファウル/Fouls,　ファウル>>に対して定義された標準的な罰則を使用しない。 +

Author: 全角スペースは滅ぼしていきましょう。
```suggestion
NOTE: このルールは<<ファウル/Fouls, ファウル>>に対して定義された標準的な罰則を使用しない。 +
```

This rule does not use the standard sanctions defined for <<ファウル/Fouls, fouls>>.

キーパー以外のロボットは、自陣のディフェンスエリア外にとどまるよう最善を尽くさなければならない。
このルールの違反は非スポーツマン行為とみなされ得る。 +
Robots other than the keeper must maintain best-effort to fully stay outside the own defense area.
Infraction of this rule can be rated as unsporting behavior.

キーパー以外のロボットが自チームのディフェンスエリアに完全に入った状態でボールに触れた場合、試合は中断され、相手チームの<<ペナルティーキック/Penalty Kick, ペナルティーキック>>で試合を再開する。
ファウルのカウンターは増加 *しない*。 +
If a robot other than the keeper touches the ball while being entirely
inside its own defense area, the game is stopped and a <<ペナルティーキック/Penalty Kick, 
penalty kick>> is awarded to the other team. The foul counter is *not*
increased.

===== 境界線との交差/Boundary Crossing
ロボットは、フィールド境界を越えてフィールド外に出るようにボールを蹴ってはならない。 +
A robot must not kick the ball over the field boundary such that the ball leaves the field.

===== キーパーによるボール保持/Keeper Held Ball
ボールはディヴィジョンAであれば5秒、ディヴィジョンBであれば10秒を超えて
<<ディフェンスエリア/Defense Area, デイフェンスエリア>>に留まってはならない。 +
The ball must not be kept in the <<Defense Area, defense area>> for more than
5 seconds (Division A) or 10 seconds (Division B).

===== ドリブルの超過/Excessive Dribbling

> ロボットは<<ドリブルデバイス/Dribbling Device, ドリブル>>が開始されたボールの位置から直線距離で測定して1m以上ドリブルしてはいけない。(ドリブルをした距離の測定方法としては)ロボットはボールと接触したときにドリブルを開始し、ボールとロボットの間に視認できるだけの間隔がある場合にドリブルを停止する。 +

Author: ```suggestion
ロボットは<<ドリブルデバイス/Dribbling Device, ドリブル>>が開始されたボールの位置から直線距離で測定して1m以上ドリブルしてはいけない。(ドリブルをした距離の測定方法としては)ロボットはボールと接触したときにドリブルを開始し、ボールとロボットの間に視認できるだけの間隔がある場合にドリブルを停止したものと見なす。 +
```

==== アウトオブプレイ中のファウル/Fouls While Ball Out Of Play
このセクションにあるファウルはボールが<<インプレイとアウトオブプレイ/Ball In And Out Of Play, アウトオブプレイ>>の時にのみ発生する。 +
Fouls in this section can only occur when the ball is <<Ball In And Out Of Play, out of play>>.

各ファウルには、再度ファウルとなるまでチームごとに2秒の猶予時間がある。 +
Each foul has a grace period of 2 seconds per team until it is raised again.

NOTE: 複数のロボットが同じファウルを2秒以内に行った場合、最初のファウルのみが計上される。 +
If multiple robots commit the same foul within 2 seconds, only the first foul counts.

NOTE: ロボットがファウルを続ける場合、猶予時間後に再び罰則が与えられる。 +
If a robot keeps committing a foul, it will be punished again after the grace period.

===== ディフェンダーのボールへの極端な接近/Defender Too Close To Ball
相手チームの<<キックオフ/Kick-Off,  キックオフ>>、<<フリーキック/Free Kick, フリーキック>>の間、ロボットはボールから少なくとも0.5m以上離れなければならない。
ファウルの前に発行されたコマンドと同じコマンドで試合が再開される。 +
A robot's distance to the ball must be at least 0.5 meters during an opponent <<キックオフ/Kick-Off, kick-off>> or <<フリーキック/Free Kick, free kick>>.
While the foul is being committed, the timer of the opponent team for bringing the ball into play is reset.

> <<主審/Referee, 人間の主審>>は、著しい妨害を受けた<<キックオフ/Kick-Off, キックオフ>>もしくは<<フリーキック/Free Kick, フリーキック>>を再度行う事ができる。 +

Author: ```suggestion
<<主審/Referee, 人間の主審>>は、著しい妨害を受けた<<キックオフ/Kick-Off, キックオフ>>もしくは<<フリーキック/Free Kick, フリーキック>>をやり直す事ができる。 +
```

The <<Referee, human referee>> may decide to repeat the <<Kick-Off, kick-off>> or <<Free Kick, free kick>> on significant disturbances.

> NOTE: <<停止/Stop, stop>>中は、ボールに近すぎる事に対する自動的な罰則はない。主審はチームが必要な距離を守っていない場合、<<イエローカード/Yellow Card, イエローカード>>を発行することで非スポーツマン行為を罰することができる。詳しい説明は「<<停止/Stop, 停止>>」を参照する事。 +

Author: stopはストップゲームか停止かストップのいずれかに統一しますが、ほかにも同様の単語・用語があるため今はスキップして #26 で対処します。

===== ストップ中のロボットの速度/Robot Stop Speed
ロボットはstop中は1.5m/s以上で動いてはいけない。このルールの反則は<<停止/Stop, ストップゲーム>>1回につき1台のロボットに対してカウントされる。 +
A robot must not move faster than 1.5 meters per second during <<停止/Stop, stop>>. A violation of this rule is only counted once per robot and stoppage.

ロボットが減速する猶予時間は2秒である。 +
There is a grace period of 2 seconds for the robots to slow down.

NOTE: このルールは<<ボール配置/Ball Placement, ボール配置>>には適用されない。 +
This rule does not apply to <<ボール配置/Ball Placement, ball placement>>.

NOTE: ロボットの速度制限の意図は、Stopコマンドが手動のボール配置と<<ロボットの交代/Robot Substitution, ロボットの交代>>に使用されるため、ロボットがフィールド内にいる人間の怪我を防ぐためである。 +
Since the stop command is used for manual ball placement and
<<ロボットの交代/Robot Substitution, robot substitution>>, the intention of the robot
speed limit is to avoid robots harming the people on the field.

===== ボール配置への干渉/Ball Placement Interference
<<ボール配置/Ball Placement, ボール配置>>の間、配置を担当しないチームのすべてのロボットはボールと配置位置の間のラインから少なくとも0.5mは離れなければならない(この領域はスタジアム状の形になる)。 +
During <<ボール配置/Ball Placement, ball placement>>, all robots of the non-placing team have to keep at least 0.5 meters distance to the line between the ball and the placement position (the forbidden area forms a stadium shape).

> ボール配置を担当しないチームのロボットが2秒以上に亘ってボールと配置場所の間に接近した場合、ファウルとなる。この場合、ボール配置を行える時間が10秒だけ追加される。 +

Author: 上の行でこう書いているので統一しましょう。

```suggestion
ボール配置を担当しないチームのロボットが2秒以上に亘ってボールと配置位置の間のラインに接近した場合、ファウルとなる。この場合、ボール配置を行える時間が10秒だけ追加される。 +
```
